packages/components/src/components/icon/common/ic-thumbs-up.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="17" viewBox="0 0 16 17" xmlns="http://www.w3.org/2000/svg">
+<path fill="var(--fill-color1)" d="M12.9915 6.11853H9.99361V3.65145C10.044 3.17774 9.90168 2.70367 9.59809 2.33354C9.2945 1.96341 8.85446 1.72753 8.37476 1.67779C8.24863 1.66299 8.12116 1.66299 7.99503 1.67779H7.21059C7.04358 1.68231 6.8828 1.74145 6.7535 1.84592C6.62419 1.9504 6.53365 2.09432 6.4961 2.25509V2.3143L5.99645 5.44255L4.22771 8.0922H1V15H10.4933C14.2106 15 14.9901 12.8537 14.9901 11.0527V8.0922C15.0404 7.61848 14.8981 7.14441 14.5945 6.77428C14.2909 6.40415 13.8509 6.16827 13.3712 6.11853C13.2451 6.10373 13.1176 6.10373 12.9915 6.11853ZM3.99787 14.0132H1.99929V9.07903H3.99787V14.0132ZM13.9908 11.0527C13.9908 13.0708 12.8816 14.0132 10.4933 14.0132H4.99716V8.73364L6.99574 5.80768L7.36048 3.15804L7.43043 2.66462H7.99503C8.71452 2.66462 8.99432 2.94093 8.99432 3.65145V7.10536H12.9915C13.711 7.10536 13.9908 7.38168 13.9908 8.0922V11.0527Z"/>
+</svg>
```

Comment:
You need to run Svgo and minify all the svg files

Reply:
ok. I'll minify the file